usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/clients/GoogleCalendarManagerClient.java

```diff
@@ -0,0 +1,176 @@
+package org.pesmypetcare.usermanagerlib.clients;
+
+import com.google.gson.Gson;
+
+import org.json.JSONObject;
+import org.pesmypetcare.usermanagerlib.datacontainers.EventData;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * @author Marc Simó
+ */
+public class GoogleCalendarManagerClient {
+    private static final String BASE_URL = "https://pes-my-pet-care.herokuapp.com/calendar/";
+    private static final String EVENT = "event/";
+    private static final String POST = "POST";
+    private static final String GET = "GET";
+    private static final String DELETE = "DELETE";
+    private static final String PUT = "PUT";
+    private static Gson GSON = new Gson();
+    private TaskManager taskManager;
+
+    public GoogleCalendarManagerClient() {
+        taskManager = new TaskManager();
+    }
+
+    /**
+     * Creates a Secondary Google Calendar in the account specified by the accessToken.
+     * @param accessToken oauth2 token needed to access the Google Calendar
+     * @param owner Name of the owner of the pet
+     * @param petName Name of the pet the calendar is created for
+     * @return The response code
+     * @throws ExecutionException When the retrieval fails
+     * @throws InterruptedException When the retrieval is interrupted
+     */
+    public int createSecondaryCalendar(String accessToken, String owner, String petName)
+        throws ExecutionException, InterruptedException {
+        taskManager = taskManager.resetTaskManager();
+        taskManager.setTaskId(POST);
+        StringBuilder response = taskManager.execute(BASE_URL + owner + "/" + petName, accessToken).get();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "/" appears 7 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5405224)